monacaのAndroid Platform v8.1.0でビルドエラーについて

## 不具合の概要(Description of bug)

- SDKのバージョン(SDK version): v3.0.7
- 不具合が発生した環境(Environment requirement)
monaca の「ビルド環境設定」にて、Android 環境の「Platform v8.1.0」を選択
- 不具合を再現させる手順・サンプルコード(Step for Reproduce bug / Sample code)
monacaで「ビルド＞Androidアプリをビルド/Build App for Android」を選択し、「Start Build」をします。
- エラーメッセージ(Error messages)
以下のようにビルドエラーが発生しています。
```
FAILURE: Build failed with an exception.
* Where:
Build file '/tmp/download/platforms/android/app/build.gradle' line: 119
* What went wrong:
A problem occurred evaluating project ':app'.
> For input string: "android-26"
```
- 原因の可能性
プラグイン内部はまだandroid-26が指定されているため、android 28にてエラー発生しています。
#88 と合わせ対応を検討進めます

--- a/ncmb.gradle
+++ b/ncmb.gradle
@@ -21,5 +21,3 @@ dependencies {
     implementation 'com.google.code.gson:gson:2.3.1'
     implementation 'com.android.support:appcompat-v7:26.1.0'
 }
-
-ext.cdvCompileSdkVersion = 'android-26'

